interface/docker-based-rock-ons/immich.rst

```diff
@@ -0,0 +1,352 @@
+#. Set-up the same external libraries for each user: causes unnecessary file metadata duplication.
+
+Here is how you can import your files into Immich.
+
+Prerequisite is that you have made your share available to Immich. See: :ref:`immich_mounting_external_library`. 
+Also note that every external library must have an owner. If you haven't created your user account(s), do so now. See: :ref:`immich_new_user`
+
+Click on your profile image on the top-right corner of the page, then click on **Administration**.
+                                   
+.. image:: /images/interface/docker-based-rock-ons/immich_administration.png
+   :width: 40%
+   :align: center
+
+ 
+Click on the **External Libraries** item in the left-side menu.
+                                   
+.. image:: /images/interface/docker-based-rock-ons/immich_external_libraries.png
+   :width: 40%
+   :align: center
+
+
+Click on the **Create library** link below your profile image on the top-right corner of the page.
+                                   
+.. image:: /images/interface/docker-based-rock-ons/immich_create_library.png
+   :width: 40%
+   :align: center
+
+
+Choose a user that will be the owner of the external library. This means that 
+all files from the external library will be imported to the selected user's library. 
+The user may choose to 
+`share their library with other users with some limitations <https://docs.immich.app/features/partner-sharing/>`_.
+
+Click on the **Create** Button.
+
+.. image:: /images/interface/docker-based-rock-ons/immich_external_library_owner.png
+   :width: 50%
+   :align: center
+
+Input the path to the external library that you've defined at :ref:`immich_mounting_external_library`. 
+E.g. ``/mnt2/MyPrivatePhotoCollection``, ``/data/test1``...
+
+Click on the **Add** button.
+
+.. image:: /images/interface/docker-based-rock-ons/immich_external_library_import.png
+   :width: 50%
+   :align: center
+
+Click on the **Validate** button to check if the external library is correctly configured.
+
+.. image:: /images/interface/docker-based-rock-ons/immich_external_library_validate.png
+   :width: 30%
+   :align: center
+
+In case or problems, please refer to
+`in-depth explanation and troubleshooting information <https://docs.immich.app/features/libraries>`_.
+
+.. _immich_other_settings:
+
+Other important settings
+------------------------------
+
+Depending on your hardware specifications and cooling capabilities, 
+you will want to consider changing the default
+`concurrency settings <https://docs.immich.app/administration/jobs-workers#jobs>`_
+when new files are being ingested into your Immich instance.
+
+Try importing a large (10k+ files) external library, then follow the system temperature and responsiveness.
+
+If no overheating is happening after 20 minutes or so, then you're done!
```

Comment:
```suggestion
If no sustained high temperatures are seen after 20 minutes or so, then you're done!
```
@kanecko Just a re-word as no correctly cooled/built machine will ever overheat - but it will show high temperatures if stressed.